deployment/terraform/iam.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_iam_policy_document" "ecs_task_execution_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+
+    actions = [
+      "sts:AssumeRole",
+    ]
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution_role" {
+  name               = "ecsTaskExecutionRole${var.environment}"
```

Comment:
In other resources, the environment comes right after the prefix vs. at the end (e.g., `ecs${var.environment}TaskExecutionRole`).